float64[] position_1
bool isReady_1
